R/use_r_workflows.R

```diff
@@ -311,6 +323,14 @@ use_doc_and_style_r <- function(workflow_name = "call-doc-and-style-r.yml",
   gha <- gha[-build_trigger_rm_lines]
   # add new build trigger
   gha <- append(gha, build_trigger_lines, after = insert_line)
+
+  if (build_trigger == "pr_comment" ) {
```

Comment:
```suggestion
  if (build_trigger == "pr_comment") {
```
there is an extra space before the closing round bracket that I removed.

Reply:
thanks!